apps/web/test/lib/getSchedule.test.ts

```diff
@@ -991,7 +991,7 @@ describe("getSchedule", () => {
           availableSlotsInTz.push(dayjs(timeObj.time).tz(Timezones["+5:30"]));
         });
       }
-      expect(availableSlotsInTz.filter((slot) => slot.format().startsWith(plus2DateString)).length).toBe(23); // 2 booking per day as limit, only one booking on that
+      expect(availableSlotsInTz.filter((slot) => slot.format().startsWith(plus2DateString)).length).toBe(24); // 2 booking per day as limit, only one booking on that
```

Comment:
@Udit-takkar It should be 23 though because there should already be one booking at that day. I missed that when changing it to 24, looking now